onprc_ehr/resources/schemas/onprc_ehr.xml

```diff
@@ -998,33 +998,250 @@
+            <column columnName="Created" />
+            <column columnName="OrderStatus" />
+            <column columnName="SavedIdentifier" />
+            <column columnName="BarcodeContent" />
+            <column columnName="CurrentBatchId" />
+            <column columnName="AlternateIdentifier" />
+            <column columnName="PrintStatus" />
+            <column columnName="ItemStatus" />
+            <column columnName="FreeTextNotes" />
+        </columns>
+    </table>
+
+    <table tableName="Prima_CaseBase" tableDbType="TABLE">
+        <columns>
+            <column columnName="Id" />
+            <column columnName="DifferentialDiagnosisId" />
+            <column columnName="PathologistId" />
+            <column columnName="PriorityLevelId" />
+            <column columnName="ResidentPathologistId" />
+            <column columnName="SerialNumber" />
+            <column columnName="SurgeryDate" />
+            <column columnName="SurgicalWheelId" />
+            <column columnName="Created" />
+            <column columnName="ResearcherId" />
+            <column columnName="StudyId" />
+            <column columnName="Discriminator" />
+            <column columnName="StudyPhaseId" />
+            <column columnName="CohortId" />
+            <column columnName="SavedIdentifier" />
+            <column columnName="Status" />
+            <column columnName="AlternateIdentifier]" />
+        </columns>
+    </table>
+
+    <table tableName="Prima_CassetteEvents" tableDbType="TABLE">
+        <columns>
+            <column columnName="Id" />
+            <column columnName="CassetteBaseId" />
+            <column columnName="EventType" />
+            <column columnName="Status" />
+            <column columnName="Trigger" />
+            <column columnName="UserId" />
+            <column columnName="Created" />
+            <column columnName="EventAction" />
+            <column columnName="TissueProcessorId" />
+            <column columnName="TissueProcessorProgramId" />
+            <column columnName="Discriminator" />
+            <column columnName="CassetteBatchId" />
+            <column columnName="CassetteOrderId" />
+            <column columnName="AutomatedCassetteArchivalMachineId" />
+            <column columnName="DisposalReasonId" />
+            <column columnName="ShipmentId" />
+            <column columnName="IsEstimated" />
+            <column columnName="PrintCount" />
+            <column columnName="BarcodeContent" />
+
+        </columns>
+    </table>
+
+    <table tableName="Prima_CassetteEventLocations" tableDbType="TABLE">
+        <columns>
+            <column columnName="CassetteEventId" />
+            <column columnName="LabStationTypeId" />
+            <column columnName="LocationId" />
+            <column columnName="WorkstationId" />
+            <column columnName="Created" />
+            <column columnName="PersonId" />
+    </table>
```

Comment:
This is missing the closing </column>, and is causing a variety of test failures. Your dev machine should be writing the same errors to the console in the debugger or in labkey.log.